aws_lambda_powertools/utilities/data_classes/common.py

```diff
@@ -103,6 +103,14 @@ def headers(self) -> Dict[str, str]:
     def query_string_parameters(self) -> Optional[Dict[str, str]]:
         return self.get("queryStringParameters")
 
+    @property
+    def resolved_query_string_parameters(self) -> Optional[Dict[str, str]]:
+        """
+        This property determines the appropriate query string parameter to be used
+        as a trusted source for validating OpenAPI.
```

Comment:
```suggestion
        This property determines the appropriate query string parameter to be used
        as a trusted source for validating OpenAPI.
        
        This is necessary because different resolvers use different formats to encode multi query string parameters.
```

Reply:
Done